kafka-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/kafka/autoconfig/KafkaAutoConfiguration.java

```diff
@@ -224,6 +235,61 @@ public StreamableKafkaMessageSource<String, byte[]> streamableKafkaMessageSource
                                            .build();
     }
 
+    @Bean
```

Comment:
:thinking:  this feels..off. The Kafka auto-config shouldn't be the point to add the Axon Server auto-config.

Just a guess, but it might have something to do with the fact Stefan (Dragisic) separated the Axon Server auto-configuration classes into two separate classes.
More specifically, the beans you're adding would only come from the `AxonServerAutoConfiguration` class, and not the `AxonServerBusAutoConfiguration` class.

Reply:
True, I would like to remove the second commit before merging. But it was needed to have the tests run correctly.